ui/src/containers/Header/index.tsx

```diff
@@ -82,6 +131,94 @@ const Header: React.FC = observer(() => {
           </TextContent>
         </Grid>
       </Modal>
+
+      <Modal
+        title="Instructions to add a new Resource in Hub"
+        isOpen={isResInstModallOpen}
+        onClose={() => setResInstModalOpen(false)}
+        width={'50%'}
+      >
+        <Grid>
+          <TextContent>
+            <Text component={TextVariants.h3}>
+              Add a new Resource through{' '}
+              <Text
+                component={TextVariants.a}
+                target="_blank"
+                href="https://github.com/tektoncd/catalog"
+              >
+                Tekton Catalog{' '}
+              </Text>
+              and needs to follow these steps
```

Comment:
What does `these steps` refer here ?? 🤔 
May be we can change it to `and follow the below steps`